doc/manual/metaprogramming.rst

```diff
@@ -45,19 +45,29 @@ each string into an object called an expression, represented by the Julia type
 
     julia> typeof(ex1)
     Expr
+    
 
 :obj:`Expr` objects contain three parts:
 
-- a ``Symbol`` identifying the kind of expression. A symbol is an
-  `interned string <https://en.wikipedia.org/wiki/String_interning>`_
+.. doctest::
+
+    julia> fieldnames(Expr)
+    3-element Array{Symbol,1}:
+     :head
+     :args
+     :typ 
+
+- `head` -- a ``Symbol`` identifying the 
+  `kind of expression <http://docs.julialang.org/en/release-0.5/devdocs/ast/#expr-types>`_. 
+  A symbol is an `interned string <https://en.wikipedia.org/wiki/String_interning>`_
```

Comment:
Oh good, I hit a speed bump with that term as well (however as I'm learning a lot of new words with Julia I tend to assume every term is correct). I feel like the word 'Token' is begging to make some appearance.

I get it I think, it's actually really simple.  A Symbol is a meta-variable.  Just as a variable is used to reference and manipulate a value in programming, so a symbol is used to reference and manipulate a variable in metaprogramming.

Would it be acceptable to link to that (or any other) SO page from the doc? (I don't think SO posts are going anywhere).